dangerfile.ts

```diff
@@ -117,7 +117,7 @@ const newComponentContainsClass = createdFiles.filter((filePath: string) => {
 });
 
 if (newComponentContainsClass.length > 0) {
-  fail(
+  warn(
```

Comment:
Agreed on Slack to change to `warn` as this will become an issue when we migrate components to TS